packages/demand-bloc-payment/src/types.ts

```diff
@@ -52,6 +62,7 @@ export type BraintreeProviderOptions = {
     hostedFieldsConfig: HostedFieldFieldOptions
     hostedFieldsStyles: Record<string, Record<string, string>>
   }
+  with3DSecure?: boolean
```

Comment:
can we name it `withThreeDSecure` ? just to match property name below

Reply:
fixed